Cargo.toml

```diff
@@ -35,6 +35,7 @@ serde_json = "1.0.89"
 schemars = "0.8.10"
 sha3 = { version = "0.10.8", default-features = false, features = [] }
 signature-verifier-api = { version = "^0.1.0", path = "packages/signature-verifier-api" }
+cast = "0.3.0"
```

Comment:
do we need both `cast` and `num-traits`?

Reply:
oh I didn't know we had `num-traits`. Let me check.